docker-images/stunnel-zookeeper/scripts/stunnel_config_generator.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+	      
+CERTS=/etc/stunnel/certs/
+
+echo "foreground = yes"
+#echo "pid=/tmp/stunnel.pid"
+echo "debug = info"
+ 
+declare -a PORTS=("2888" "3888")
+for port in "${PORTS[@]}"
+do
+NODE=1
+ while [ $NODE -le $ZOOKEEPER_NODE_COUNT ]; do
+	        
+		PEER=${BASE_HOSTNAME}-$((NODE-1))
+		KEY_AND_CERT=${CERTS}${PEER}	
+		PEM=/tmp/${PEER}.pem
+
+		# Not necessarily the way to create pem file, might have to use openssl
+		cat ${KEY_AND_CERT}.crt ${KEY_AND_CERT}.key > ${PEM}
+		chmod 640 ${PEM}
+
+		# Stunnel client configuration
+		cat <<-EOF
+		[${PEER})]
+		client=yes
+		cert=${PEM}
+		accept=127.0.0.1:$(expr $port \* 10 + $NODE)
+		connect=${PEER}.${BASE_FQDN}:$port
+
+		EOF
+		let NODE=NODE+1
+  done
+	# Zookeeper port where stunnel forwards recieved traffic
+	CONNECTOR_PORT=$(expr $port \* 10)
```

Comment:
Using the 28880 and 38880 ports (which is what I set in the CO as well) shifts the listeners ports of stunnel to 28881 for zookeeper-0, 28882 for zookeeper-1 and so on. If we want to stick with the rule that port = 2888 * 10 + index (where index starts from 0), we should change the 28880 and 38880 ports. What do you think about something like 2999 and 3999 ? @kyguy @tombentley @scholzj